influxql/query_executor.go

```diff
@@ -66,6 +66,9 @@ type ExecutionOptions struct {
 
 	// Node to execute on.
 	NodeID uint64
+
+	// Do not log the normalized statement for this query.
+	Quiet bool
```

Comment:
Like @jsternberg mentioned, disabling query logs per service would be a larger change probably best left for post-1.0.